### PR TITLE
Version2

### PR DESCRIPTION
**Pull request by ryanmitchell**
Version 2 here we go

### CODE REVIEW

**Comment by roydeanjr** [comment]
Did you update the marketplace?
I just looked at the marketplace version and it said 2.0.0 and that the extension was already installed on my install but it shows v1.1.0 or something like that.
I deleted the extension but not the data and now the extension will not install.
Any ideas?

**Comment by ryanmitchell** [comment]
The marketplace updates automatically. I suspect it didnt install as your version was probably the v2 branch?

**Comment by roydeanjr** [comment]
I reverted to an older version: v1.0.0 and the auto update signalled a new version was available, however, it reported that v1.1.0 was the new version available. I updated and that is for sure not version 2.0.0.

When I search for Kitchen in the Browse Extensions field I get that v2.0.0 is the version that is available. Yet, the system will not download and install v2.0.0. I even tried deleting the extension with delete data as the option and the progress bar displays, but does not install.

Nothing I do via the update process on the install will install v2.0.0.

**Comment by roydeanjr** [comment]
image

**Comment by roydeanjr** [comment]
image

**Comment by ryanmitchell** [comment]
Just install by uploading this version.

**Comment by ryanmitchell** [comment]
Oh wait. I didn’t tag the release - need to do that.

**Comment by roydeanjr** [comment]
How long does it take to reflect on the marketplace?

**Comment by ryanmitchell** [comment]
Seems to have updated there already.
https://tastyigniter.com/marketplace/item/thoughtco-kitchendisplay

**Comment by roydeanjr** [comment]
I attempted and it reflected a hash mismatch. I refreshed the updates screen and now it shows 2.0.0. Update completed. Thank you!

File: views/summary/view.blade.php [hunk 1]
```diff
@@ -26,6 +43,7 @@
 							<p class="w-100 text-wrap"><em>{!! $order->comment !}}</em></p>
```
Comment (BreakSecurity) [inline views/summary/view.blade.php:56]: {!! $order->comment !!}}

Missing !

Comment (roydeanjr) [reply views/summary/view.blade.php:56]: Did not even notice that! And should have been caught when merged. Now fixed and a new PR created. Thank you!

**Pull request merged**